### PR TITLE
Initial revision

### PR DESCRIPTION
**Pull request by timg236**
Import the files used for the first beta release.

### CODE REVIEW

**Comment by timg236** [comment]
@XECDesign - is this everything that you need for the APT package ?

**Comment by andrum99** [comment]
These files are missing a licence. If you are intending these to be open-source, you would need to add a suitable licence somewhere, for example in a LICENCE file in the root of the repo.

Also, would you be interested in man pages for these scripts? I can write these.

**Comment by timg236** [comment]
> These files are missing a licence. If you are intending these to be open-source, you would need to add a suitable licence somewhere, for example in a LICENCE file in the root of the repo.
> 
> Also, would you be interested in man pages for these scripts? I can write these.

A license is on the todo list. Happy to accept PR for man pages, although I'd recommend holding off until any feedback from the beta is merge in-case of interface changes. Hopefully, there won't be any though!

**Comment by XECDesign** [comment]
We could generate the man pages with help2man.

I've attached the license in Debian format. Probably not suitable for the source repo, but you can copy the appropriate bits from there.
[copyright.txt](https://github.com/raspberrypi/rpi-eeprom/files/3565528/copyright.txt)

Everything is BSD-3 except for the EEPROM images themselves, which are restricted for use with Raspberry Pi products only.

File: rpi-eeprom-update [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+
+   if [ "${USE_FLASHROM}" = 0 ]; then
+      [ -f "${RECOVERY_BIN}" ] || die "${RECOVERY_BIN} not found"
+   fi
+}
+
+usage() {
+cat <<EOF
+rpi-eeprom-update [options]... [FILE]
+   Checks whether there Raspberry Pi bootloader EEPROM is up to date and
```
Comment (andrum99) [inline rpi-eeprom-update]: there -> the. Add hyphenation: up to date -> up-to-date

Comment (timg236) [reply rpi-eeprom-update]: Will fix in next set of script updates, this is just importing the files that are already available

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,67 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: rpi-eeprom
+Source: https://github.com/raspberrypi/rpi-eeprom
+
+Files: *
+Copyright: 2019, Raspberry Pi (Trading) Ltd.
+License: BSD-3
+
+Files: lib/firmware/*
```
Comment (XECDesign) [inline LICENSE]: The path has changes since the file was written.

**Pull request merged**